analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/TokenTypeAnalyzerTest.Classifier.cs

```diff
@@ -1375,4 +1375,21 @@ public class C
                 {{fieldDeclaration}}
             }
             """, allowSemanticModel);
+
+#if NET
+
+    [TestMethod]
+    public void CSharp12Syntax_Classification() =>
+        ClassifierTestHarness.AssertTokenTypes("""
+            class PrimaryConstructor([k:int] [u:i] = [n:1])
```

Comment:
It is better to use `Int32` instead. The identifiername logic is much more error prone than the keyword logic. Same is true below.